packages/internal/src/storage-test-utils.js

```diff
@@ -1,11 +1,67 @@
 // @ts-check
 import { Far } from '@endo/far';
-import { makeMarshal } from '@endo/marshal';
+import { makeMarshal, Remotable } from '@endo/marshal';
 import { makeChainStorageRoot } from './lib-chainStorage.js';
 import { bindAllMethods } from './method-tools.js';
 
 const { Fail, quote: q } = assert;
 
+/**
+ * A convertSlotToVal function that produces basic Remotables
+ *
+ * @param {string} _slotId
+ * @param {string} iface
+ */
+export const slotToRemotable = (_slotId, iface = 'Alleged: Remotable') =>
```

Comment:
```suggestion
export const slotToRemotable = (_slotId, iface = 'Remotable') =>
```
Please do fix this, even though this is a post-merge review comment. The `Remotable` default is `'Remotable'`. The validation rule is that the must either be `'Remotable'` or must start with `'Alleged: '`. (The fact that it is a Remotable is reliable, as opposed to an unreliable potentially third-party allegation. So we don't require the prefix on this one iface string value.)

Also, please add a comment to the effect of
```js
// Assumes that all slots are Remotables, i.e., that none are promises.
```